app/views/layouts/report_only.html.haml

```diff
@@ -2,7 +2,7 @@
 %html.layout-pf.layout-pf-fixed.transitions{:lang => I18n.locale.to_s.sub('-', '_')}
   %head
     %title
-      = "%{product}: %{title}" % {:product => Vmdb::Appliance.PRODUCT_NAME, :title => @report.title}
+      = "%{product}: %{title}" % {:product => Vmdb::Appliance.PRODUCT_NAME, :title =>  @report_title}
```

Comment:
The whitespace before `@report_title` is unnecessary.

Reply:
done 👍 thanks